tests/integration/test_pip.py

```diff
@@ -73,8 +73,8 @@
         ),
         pytest.param(
             utils.TestParameters(
-                repo="https://github.com/cachito-testing/cachito-pip-without-metadata.git",
-                ref="4fb61a80f9bb19fdd422a74602b108d405ede299",
+                repo="https://github.com/cachito-testing/cachi2-pip-extra.git",
```

Comment:
I know this is a GH organization to host test repositories, but you can't just rename publicly reachable repositories with no further notice, that's both confusing and a bad practice. What's worse is that you also force pushed the main branch overwriting history for everyone who may have the repository in question checked out locally - this is a big no no! We just lost history on the following commit: https://github.com/cachito-testing/cachi2-pip-extra/commit/e5e81c7 . I understand that all this was to address the fact that the original repository was tailored to a single specific use case, but in that case such a repository needs to be properly archived and a new one needs to be created to take its place to retain whole history.